### ISSUE TITLE
Crimson cult

### ISSUE THREAD

**Issue by devtn2001** [Azanor/thaumcraft-suggestions#449]
Maybe some lore?
We have a crimson cult and eldrich. 
What if we will habe a choice to get on one side: to be like a ghost or to be crimson. In the book will be some researches for one side that you have chosen.

**Comment by Neprim** [comment]
It isn't a RPG.

**Comment by devtn2001** [comment]
Lore in the book says it has some rpg

**Comment by VadersFist0501** [comment]
You are not supposed to join a side, you are just a somewhat mad scientist who wants to know how to exploit the elements and the eldritch for power... for some reason.
Minecraft Logic: makes about as much sense as the real world.

**Comment by Crimson-Cult** [comment]
I disagree with joining a faction, but maybe having ways to make peace with the Cult, like offering a Primordial Pearl to a Praetor, so that you no longer aggro them and can then trade with him similar to how you trade with a Pech. I agree with the idea that the Crimson Cult has a lot more to offer. I go into some detail here #466
